Common/src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -71,7 +71,7 @@
   "death.attack.overheat.player.item": "%1$s, %3$s kullanan %2$s'den kaçmaya çalışırken içini fitilledi.",
 
   "power.toomanyorigins.hover.name": "Süzül",
-  "power.toomanyorigins.hover.description": "Birçok arıdan yapılmış olman, havadayken süzülebilmeni sağlar.",
+  "power.toomanyorigins.hover.description": "Birçok arıdan yapılmış olman, havada asılı kalabilmeni sağlar.",
```

Comment:
more accurate

changed hover's translation from Glide to "Stay still in the air"

Hover isn't a word in Turkish